dags/civicrm_email_report_import_pipeline.py

```diff
@@ -0,0 +1,126 @@
+# Note: DagBag.process_file skips files without "airflow" or "DAG" in them
+
+import os
+import logging
+from datetime import timedelta
+
+from data_pipeline.utils.dags.data_pipeline_dag_utils import create_dag, create_python_task
+from data_pipeline.utils.pipeline_file_io import get_yaml_file_as_dict
+from data_pipeline.civicrm_email_report.civicrm_email_report import iter_email_report
+from data_pipeline.utils.data_store.bq_data_service import (
+    get_distinct_values_from_bq,
+    load_given_json_list_data_from_tempdir_to_bq
+)
+from data_pipeline.civicrm_email_report.get_civicrm_email_report_data_config import (
+    CiviCrmEmailReportDataConfig
+)
+from data_pipeline.utils.pipeline_file_io import read_file_content
+
+LOGGER = logging.getLogger(__name__)
+
+CIVICRM_KEY_SECRET_FILE_PATH_ENV = "CIVICRM_KEY_SECRET_FILE_PATH"
```

Comment:
I believe this can now be removed? (And any other related mention in other files)